Integrations/MISP_V2/MISP_V2_test.py

```diff
@@ -71,3 +71,81 @@ def test_extract_error(mocker):
     assert err == []
 
     # TODO check errors
+
+def test_build_misp_complex_filter(mocker):
```

Comment:
please add the following test cases:
1. exclude a tag
2. if its a valid query, something like `OR:tag1;AND:tag2;OR:tag3,tag4;`

Reply:
1. already done using the new query
added for old query
2. already done in the last step

Reply:
sorry, but i must be missing something, where do you see these?

Reply:
1. ` complex_query_NOT_single = "NOT:tag1"
    expected = {'NOT': ['tag1']}
    actual = build_misp_complex_filter(complex_query_NOT_single)
    assert actual == expected`

2. my bad ...this is not a valid query.

Reply:
1. i meant for the previous query syntax, but i see you added it in the last commit
2. 👍